### ISSUE TITLE
Assigning scope attribute for table headers

### ISSUE THREAD

**Issue by alipphardt** [nicolaskruchten/pivottable#536]
First of all, thank you for this library it has been greatly helpful for display of data tables and visualizations.

Secondly, I work for the government and we are required to make data tables accessible (Section 508/WCAG2 compliant). This library already does a great job by designating thead, tbody, and table headers as th. Only missing component is the scope attribute for th tags. This would involve assigning an attribute of scope="col" for all th in a thead tag and scope="row" for all th in a tbody tag. I suspect this it not too difficult to do but I'm relatively new to both github and coffeescript.

http://webaim.org/techniques/tables/data#headers

**Comment by nicolaskruchten** [comment]
Sorry I'm just getting to this now... How should we handle `th`s that span multiple columns/rows? `scope="colgroup"` and `scope="rowgroup"` ?

**Comment by alipphardt** [comment]
From the testing that I have done, best thing is to just keep these to
scope="col" or scope="row". Attributes scope="colgroup" and
scope="rowgroup" are not widely supported among screen readers and other
assistive technologies.
